packages/back-end/src/integrations/BigQuery.ts

```diff
@@ -106,4 +109,48 @@ export default class BigQuery extends SqlIntegration {
       database
     );
   }
+  async getInformationSchema(): Promise<InformationSchema[]> {
+    const datasets = await this.runQuery(
+      `SELECT * FROM ${`\`${this.params.projectId}.INFORMATION_SCHEMA.SCHEMATA\``}`
+    );
+
+    const results = [];
+
+    for (const dataset of datasets) {
```

Comment:
@lukesonnet @jdorn I've updated this so if one or more of the queries fail, we don't throw an error and stop the whole job. This way, if for some reason a user had permission to fetch all of the datasets, but couldn't run the query for a particular dataset, we'd just skip it, log it in the error log, and keep going.

Let me know if you think this is a sufficient approach - currently, the logic for handling information schema errors is all or nothing - e.g. if there is an error, we only show the error, and only if there isn't an error do we show the `SchemaBrowser`.

I can definitely see users getting confused about why they may not be seeing all datasets, but I'm not sure if this will ever even happen, so not sure if it's worth investing the time to make this process better.